vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/VariantExporter.java

```diff
@@ -208,6 +209,13 @@ private VCFHeader addMissingMetadataLines(VCFHeader header) {
         if (header.getFormatHeaderLine("GT") == null) {
             header.addMetaDataLine(new VCFFormatHeaderLine("GT", 1, VCFHeaderLineType.String, "Genotype"));
         }
+        // CSQ line
+        if (header.getFormatHeaderLine("CSQ") == null) {
+            header.addMetaDataLine(new VCFInfoHeaderLine("CSQ", 1, VCFHeaderLineType.String,
```

Comment:
There's no evaProperties instance in this class. Should I add it via constructor or modify the method signature?